specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/settings.json

```diff
@@ -0,0 +1,208 @@
+              "schema": {
+                "$ref": "common.json#/definitions/KeyVaultError"
+              }
+            }
+          },
+          "x-ms-examples": {
+            "Create key": {
+              "$ref": "./examples/UpdateSetting-example.json"
+            }
+          }
+        },
+        "get": {
+          "tags": [
+            "Settings"
+          ],
+          "operationId": "GetSettingValue",
+          "summary": "Get specified account setting value.",
+          "description": "Retrieves the value of a specified, value account setting.",
+          "parameters": [
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "An account setting containing the name of the setting and the value it is set to.",
+              "schema": {
+                "$ref": "#/definitions/Setting"
+              }
+            },
+            "default": {
+              "description": "Key Vault error response describing why the operation failed.",
+              "schema": {
+                "$ref": "common.json#/definitions/KeyVaultError"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          },
+          "x-ms-examples": {
+            "GetKeys": {
+              "$ref": "./examples/UpdateSetting-example.json"
+            }
+          }
+        }
+      },
+      "/settings": {
+        "get": {
+          "tags": [
+            "Settings"
+          ],
+          "operationId": "GetSettings",
+          "summary": "List account settings.",
+          "description": "Retrieves a list of all the available account settings that can be configured.",
+          "parameters": [
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "A response message containing a list of account settings with their associated value.",
+              "schema": {
+                "$ref": "#/definitions/SettingsListResult"
+              }
+            },
+            "default": {
+              "description": "Key Vault error response describing why the operation failed.",
+              "schema": {
+                "$ref": "common.json#/definitions/KeyVaultError"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          },
+          "x-ms-examples": {
+            "GetKeys": {
+              "$ref": "./examples/ListSettings-example.json"
+            }
+          }
+        }
+      }
+    },
+    "definitions": {
+      "UpdateSettingsRequest": {
+        "properties": {
+          "value": {
+            "type": "string",
+            "description": "The account setting value. True if setting is enabled, false if disabled"
```

Comment:
It's a string named "value", so why would someone pass boolean values? Seems there should be a `enabled` property that should be true or false, yeah?